### PR TITLE
 gitg: Update to version 3.32.1

### PR DESCRIPTION
**Pull request by albfan**
Update gitg to 3.32.1

Added patched libdazzle dependency to access pango_font_description_to_css util

### CODE REVIEW

File: mingw-w64-gitg/PKGBUILD [hunk 1]
```diff
@@ -33,17 +33,19 @@ license=("GPL2+")
...
-}
+
+)
+
+
+
+source=(https://download.gnome.org/sources/${_realname}/${pkgver:0:4}/${_realname}-$pkgver.tar.xz)
+sha256sums=('24a4aabf8a42aa2e783e1fb5996ebb3c2a82a01b9689269f1329517ef124ef5a')
+
+
```
Comment (Alexpux) [inline mingw-w64-gitg/PKGBUILD]: Remove this many empty lines. Add libdazzle to depends if needed

Comment (albfan) [reply mingw-w64-gitg/PKGBUILD]: Yes sorry, trying to fix this afk seems that mobile do not remove lines, hehe. Done

**Comment by albfan** [comment]
This fails because libdazzle is not found. Can I add libdazzle and gitg update in same PR and they will be found and build? Or just need to launch this again when libdazzle PR #5649 is merged?

File: mingw-w64-gitg/PKGBUILD [hunk 2]
```diff
@@ -32,18 +33,9 @@ options=('strip')
...
-            'b616b32aff2d48ded0261c1fe9c6905c1adc5f11cf1422efe5baad35785ded0b'
-            '42e895d458e50bfb2cd2f23e0f8bd18a23ba33f6ce0b1ca2c0fd7f6377564080')
-
-prepare() {
-  cd "${srcdir}"/${_realname}-${pkgver}
-  patch -Np1 -i "${srcdir}/Fix-libgit2-glib-cache.patch"
-  patch -Np1 -i "${srcdir}/Config-app-as-GUI.patch"
-}
+source=(https://download.gnome.org/sources/${_realname}/${pkgver:0:4}/${_realname}-$pkgver.tar.xz)
+source=(https://download.gnome.org/sources/${_realname}/${pkgver:0:4}/${_realname}-$pkgver.tar.xz)
```
Comment (Alexpux) [inline mingw-w64-gitg/PKGBUILD]: Remove duplicate sources line

Comment (albfan) [reply mingw-w64-gitg/PKGBUILD]: Done

**Comment by Alexpux** [comment]
@albfan gitg not buildable for me:
````
[30/199] Compiling C object libgitg/5cbadd7@@gitg-1.0@sha/meson-generated_gitg-date.c.obj.
FAILED: libgitg/5cbadd7@@gitg-1.0@sha/meson-generated_gitg-date.c.obj 
cc @libgitg/5cbadd7@@gitg-1.0@sha/meson-generated_gitg-date.c.obj.rsp
libgitg/5cbadd7@@gitg-1.0@sha/gitg-date.c:31:10: fatal error: gdesktop-enums.h: No such file or directory
   31 | #include <gdesktop-enums.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[31/199] Compiling C object libgitg/5cbadd7@@gitg-1.0@sha/meson-generated_gitg-commit.c.obj.
[32/199] Compiling C object libgitg/5cbadd7@@gitg-1.0@sha/meson-generated_gitg-commit-list-view.c.obj.
````

Seemd meson not add ````gsettings-desktop-schemas```` to include path

**Comment by albfan** [comment]
In my case (local) the header is there but is failing too. I checked with vala maintainer but I'm not really sure what's happening let me check again, now at least libdazzle is merged.

**Comment by albfan** [comment]
@Alexpux I see that generated build.ninja has:

```
-IC:/msys64/mingw64/include/libdazzle-1.0" "-I/mingw64/include/gsettings-desktop-schemas" "-IC:/msys64/mingw64/include/gtksourceview-3.0"
```

so seems that meson cannot find gsettings-desktop-schemas because it do not add the `C:\`

**Comment by albfan** [comment]
@Alexpux changing  `/mingw64/share/pkgconfig/gsettings-desktop-schemas.pc`

```
prefix=/mingw64
libdir=${prefix}/lib
includedir=${prefix}/include

Name: gsettings-desktop-schemas
Description: Shared GSettings schemas for the desktop, including helper headers
Version: 3.32.0
Cflags: -I${includedir}/gsettings-desktop-schemas
```

fixes the problem. I think we need to patch the pkg-config file

**Comment by Alexpux** [comment]
@albfan I fix gsettins by https://github.com/msys2/MINGW-packages/commit/95d07e8ae04e99eed1c4c6b8a3873fe4f9395221 but get other error now:

````
C:/building/msys64/mingw64/include/libdazzle-1.0/dazzle.h:37:10: fatal error: actions/dzl-action-group.h: No such file or directory
   37 | #include "actions/dzl-action-group.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
````

**Comment by albfan** [comment]
@Alexpux I uploaded a new MR for libdazzle that fix that #5685.

**Pull request merged**